### DISCUSSION TITLE
Display Non-Markdown Files in Notes/Repository View

### DISCUSSION THREAD

**Discussion by quinn-p-mchugh** [damionx7/Zettel-Notes-Documentation#61] [Feature Requests]
Copying from issue #45 and [Canny](https://zettel-notes.canny.io/feature-requests/p/display-non-markdown-files):

> I'd like to be able to open the synchronized non-Markdown files in the native viewers I have assigned to them. Even if they shouldn't be displayed in the application GUI, at least providing access via a `com.android.documentsui` portal is enough. However, I'd like to be able to see them via the application GUI too.

From Sam (a Canny commenter):

> I too would like the option to display non-markdown/non-note files in the same hierarchy as the notes. I would recommend adding this as a setting.

Related requests: #62 and #63

**Comment by damionx7** [comment]
Closed in favor of https://github.com/damionx7/Zettel-Notes-Documentation/discussions/70
